hil/api.py

```diff
@@ -1098,14 +1104,46 @@ def port_revert(switch, port):
 
     if port.nic is None:
         raise errors.NotFoundError(port.label + " not attached")
-    if port.nic.current_action:
-        raise errors.BlockedError("Port already has a pending action.")
+    check_pending_action(port.nic)
+    unique_id = str(uuid.uuid4())
```

Comment:
just curious, why not just `uuid`?  That's not a default keyword right? I am perfectly fine with `unique_id`.

Reply:
```
>>> import uuid
>>> uuid = uuid.uuid4()
>>> print uuid
49185699-3189-4865-b65f-b2529dba3580
>>> another_uuid = uuid.uuid4()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'UUID' object has no attribute 'uuid4'
```
Just so that it doens't pollute the namespace.

Reply:
g2k 👍